code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -264,11 +264,11 @@
 
 /obj/item/robot_module/medical
 	name = "Medical"
-	basic_modules = list( //FULPSTATION MEDBORG UPGRADES by Surrealistik March 2020
+	basic_modules = list(
 		/obj/item/assembly/flash/cyborg,
 		/obj/item/healthanalyzer,
 		/obj/item/reagent_containers/borghypo,
-		/obj/item/borg/apparatus/beaker/medical, //FULPSTATION MEDBORG UPGRADES by Surrealistik March 2020

```

Comment:
?

Reply:
I mentioned that I moved all fulpstation overwrite vars to the fulpstation overwrite vars dm.  Should make easier merges

Reply:
I mentioned that I moved all fulpstation overwrite vars to the fulpstation overwrite vars dm.  Should make easier merges

Reply:
I mentioned that I moved all fulpstation overwrite vars to the fulpstation overwrite vars dm.  Should make easier merges

Reply:
ah